### DISCUSSION TITLE
Chart issue

### DISCUSSION THREAD

**Discussion by kassyrael** [vikramlearning/blazorbootstrap#815] [Q&A]
Hi, I'm trying to use the chart from your bootstrap. The first run is working fine.
![chart running](https://github.com/user-attachments/assets/13f3f6e2-f920-45b6-a7e9-de7850139261)

but when I tried to rerun it, it doesn't work anymore. I did not change any code.
![chart not running](https://github.com/user-attachments/assets/de3a9e69-1c85-4b79-9fb5-0950e95ffc56)

your answer will be a great help. thanks

**Comment by gvreddy04** [comment]
@kassyrael Thank you for trying BlazorBootstrap. Please share a sample code or GitHub repo to reproduce the issue with minimal steps.

### Versions (please complete the following information):

- .NET Version: [e.g. .NET 6, .NET 7, .NET 8]
- BlazorBootstrap: [e.g. 2.0.0, 2.1.0, 2.2.0]
- Blazor WebAssembly / Server: [e.g. WebAssembly, Server]
- Blazor Interactive Render Mode: [e.g. Auto, Server, WebAssembly, None]
- Blazor Interactivity Location: [e.g. Global, Per page/component]

Also, check and confirm that the demos are working on your end.
Demos link: https://demos.blazorbootstrap.com/charts/bar-chart

**Reply by kassyrael** [reply]
@gvreddy04 

I'm using blazorbootstrap 2.2.1 version and the blazor settings below
![image](https://github.com/user-attachments/assets/11e97c04-84f3-4e3f-9113-3df09327826b)


please see the sample project here

https://drive.google.com/file/d/1hQcG9X2RQqpe1wazGuDg_Nrwr3t00_V4/view?usp=drive_link

thanks
